contracts/src/Teleporter/upgrades/TeleporterUpgradeable.sol

```diff
@@ -128,6 +134,34 @@ abstract contract TeleporterUpgradeable is ITeleporterReceiver {
         emit TeleporterAddressPaused(teleporterAddress);
     }
 
+    /**
+     * @dev Unpauses a Teleporter address from interacting with this contract.
+     * After unpausing a Teleporter address, it will again be able to deliver messages
+     * to this contract, and this contract can send messages through that Teleporter address.
+     * The address does not need to be registered with the Teleporter registry.
+     * Emits a {TeleporterAddressUnpaused} event if successfully unpaused.
+     * Requirements:
+     *
+     * - `msg.sender` must have Teleporter upgrade access.
+     * - `teleporterAddress` is not the zero address.
+     * - `teleporterAddress` is already paused.
+     */
+    function unpauseTeleporterAddress(
+        address teleporterAddress
+    ) public virtual {
+        _checkTeleporterUpgradeAccess();
+        require(
+            teleporterAddress != address(0),
+            "TeleporterUpgradeable: zero Teleporter address"
+        );
+        require(
+            isTeleporterAddressPaused(teleporterAddress),
+            "TeleporterUpgradeable: address not paused"
```

Comment:
I left a similar comment on https://github.com/ava-labs/teleporter/pull/185, but is this requirement necessary? I suppose it doesn't matter that much since both the pause and unpause methods are only callable by privileged entities.

Reply:
same as previous comment, would prefer to leave as is for better indication of when an address is already paused, and saves gas for redundant call.